src/lavinmq/http/controller.cr

```diff
@@ -191,38 +191,35 @@ module LavinMQ
       end
 
       private def refuse_unless_vhost_access(context, user, vhost)
+        return if user.tags.any? &.administrator?
         unless user.permissions.has_key?(vhost)
           @log.warn { "user=#{user.name} does not have permissions to access vhost=#{vhost}" }
           access_refused(context)
         end
       end
 
       private def refuse_unless_management(context, user, vhost = nil)
-        unless user.tags.any? { |t| t.management? || t.administrator? || t.policy_maker? || t.monitoring? }
+        unless user.tags.any?
```

Comment:
is this correct? what about the `impersonator` tag? that shouldn't give management access?

Reply:
Looked for the pre-defined tags we have https://github.com/cloudamqp/lavinmq/blob/main/src/lavinmq/user.cr#L6-L10
initialize and create user are limited to those.

Reply:
Yes, but any tag should be able to be added to a user, not just the predefined.

Reply:
Use module and constants instead of enum? Then any tag can be set but still keep pre-defined. https://gist.github.com/tbroden84/e1c004dd23462fe76582eaf1792773e7